src/main/java/seedu/duke/Task.java

```diff
@@ -0,0 +1,81 @@
+package seedu.duke;
+
+/**
+ * Represents a task.
+ * A <code>Task</code> object corresponds to a task with a command that starts
+ * with "todo", "event" or "deadline", followed by description.
+ *
+ */
+public class Task {
+    private String description;
+    boolean isDone;
+    int doneIcon;
+
+    /**
+     * Constructor of the Task class.
+     *
+     * @param description the description of the task
+     */
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+        doneIcon = 0;
+    }
+
+    /**
+     * Return the status icon depending on whether the task is marked done or not.
+     *
+     * @return the status icon
+     */
+    public String getStatusIcon() {
+        return (isDone ? "[✓]" : "[✗]");
+    }
+
+    /**
+     * Sets the isDone variable to true and the doneIcon to 1.
+     * Prints the done message.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+        this.doneIcon = 1;
```

Comment:
Maybe you can reuse the `updateDone` function here to avoid repetitive code?

Reply:
Noted, I will update it soon.